CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Bugfixes
 
+- [#6413](https://github.com/influxdata/influxdb/pull/6413): Prevent goroutine leak from persistent http connections.
```

Comment:
Add it to the bottom and also include a `Thanks @aaronnister.` to the end so you get credit for your work.